gradle/libs.versions.toml

```diff
@@ -37,7 +37,7 @@ wiremock = "2.27.2"
 # Versions which start with managed- are managed by Micronaut in the sense
 # that they will appear in the Micronaut BOM as <properties>
 #
-managed-commons-dbcp = "2.9.0"
+managed-commons-dbcp = ""
```

Comment:
Shoudl this be left to 2.9.0 or an we remove it https://github.com/micronaut-projects/micronaut-sql/blob/master/gradle/libs.versions.toml#L7